library/src/main/java/hotchemi/android/rate/UriHelper.java

```diff
@@ -13,4 +18,14 @@ static Uri getGooglePlay(String packageName) {
         return packageName == null ? null : Uri.parse(GOOGLE_PLAY + packageName);
     }
 
+    static boolean isPackageExists(Context context, String targetPackage) {
+        List<ApplicationInfo> packages;
```

Comment:
Please fix as following code.

```java
PackageManager pm = context.getPackageManager();
List<ApplicationInfo> packages = pm.getInstalledApplications(0);
```

And please tell me what `getInstalledApplications(0)` means(flag)?